src/components/accounts/ui/buttons/ColorToggleBtn.js

```diff
@@ -16,6 +16,8 @@ export const ColorToggleButton = ({ value, options, handleAdd }) => {
         return 'Yes';
       case '0':
         return 'No';
+      case '2':
```

Comment:
I have concern about this, as 0,1,2 mean different things, please show me the data which is being returned. The value of the label may be in the data